sale_order_global_stock_route/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2020 Tecnativa - Carlos Roca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Sale Order Global Stock Route',
+    'summary': 'Add the possibility to choose '
+               'one warehouse path for an order',
+    'version': '12.0.1.0.0',
+    'development_status': 'Alpha',
```

Comment:
Why alpha? Shouldn't this be `beta` once at least 2 of you have reviewed it?

Reply:
Change done